### DISCUSSION TITLE
add custom g-code not working ?

### DISCUSSION THREAD

**Discussion by Tonino84** [OrcaSlicer/OrcaSlicer#1258] [General]
Hello guys,

I'm trying to add custom g-code from preview view; if I right click on the layers bar, I can see it but ... it's disabled!
Version 1.6.3

Thanks

**Comment by Tonino84** [comment]
I found that it turn off, when I choose print by object and not by layer..... but why?

**Comment by olaab** [comment]
I also try to add custom Maskin stop G-code but nothin happens. I can see it!
What to solve this?
![0057](https://github.com/SoftFever/OrcaSlicer/assets/32199168/517b989d-5905-46e3-ad6c-997549ef5964)
![0058](https://github.com/SoftFever/OrcaSlicer/assets/32199168/206595e0-11fb-4baf-ac4e-745f5239bbc0)

**Comment by Staubgeborener** [comment]
I can confirm this
